qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -0,0 +1,125 @@
+            migration_bg_kill_command = pkill -9 tcpdump
+            kill_vm_on_error = yes
+            iterations = 2
+            used_mem = 1024
+            mig_timeout = 3600
+            ping_pong = 1
+            variants:
+                - tcp:
+                    migration_protocol = "tcp"
+                - unix:
+                    migration_protocol = "unix"
+                    no Host_RHEL.5
+                - exec:
+                    migration_protocol = "exec"
+                    variants:
+                        - @default_exec:
+                        - gzip_exec:
+                            migration_exec_cmd_src = "gzip -c > %s"
+                            migration_exec_cmd_dst = "gzip -c -d %s"
+                - fd:
+                    migration_protocol = "fd"
+        - reboot_after_hotplug:
+            post_test_type = boot
+            post_test_name = reboot
+            reboot_method = shell
+            kill_vm_on_error = yes
+            login_timeout = 240
+            reboot_count = 1
+        - system_reset_after_hotplug:
+            post_test_type = boot
+            post_test_name = system reset
+            reboot_method = system_reset
+            sleep_before_reset = 0
+    variants:
+        # Notes:
+        #     Unplug memory device doesn't support by Windows Guest
+        #     For linux guest, unplug always failed with error memory
+        #     still in used. And it will fix in later by linux kernel.
+        - unplug_pluged_mem:
+            unplug_mem = yes
+            no Windows
+        - unplug_booted_mem:
+            mem_devs += " mem3"
+            unplug_mem = mem3
+            no Windows
+        - no_unplug_mem:
+            unplug_mem = no
+    variants:
+        - backend_ram:
+            backend_mem = memory-backend-ram
+            hotplug_mem = mem2
+            size_mem_mem2 = 1G
+        - backend_file:
+            backend_mem = memory-backend-file
+            setup_hugepages = yes
+            # mem path should be the hugpage path
+            mem-path = /mnt/kvm_hugepage
+            hotplug_mem = mem2
+            size_mem_mem2 = 1G
+    variants:
+        - one_numa_node:
+           guest_numa_nodes = "node0"
+           del numa_mem
+           del numa_cpus
+           del numa_nodeid
+        - two_numa_nodes:
+           guest_numa_nodes = "node0 node1"
+           del numa_mem
+           del numa_cpus
+           numa_nodeid = 0
+           node_dimm_mem2 = 0
+           node_dimm_mem1 = 1
+           numa_nodeid_node0 = 0
+           numa_nodeid_node1 = 1
+    variants:
+        - policy_default:
+           policy_mem = default
+        - policy_bind:
+           policy_mem = bind
+           host-nodes = 0
+        - policy_interleave:
+           policy_mem = interleave
+           host-nodes = 0
+        - policy_preferred:
+           policy_mem = preferred
+           host-nodes = 0
+        - no_policy:
+           del policy_mem
```

Comment:
Confirmed with feature owner, offline base migration scenario has cut and and windows guest only support hotplug and only latest RHEL guest support hotplug & unplug(still have bug) now. 
And I have updated configuration file, please retest now.